packages/manager/modules/bm-server-components/src/server-dashboard/dashboard.html

```diff
@@ -108,7 +108,9 @@
                 go-to-resiliation="$ctrl.goToResiliation()"
                 service-infos="$ctrl.serviceInfos"
                 highlight-engagement="$ctrl.server.shouldReengage"
-                tracking-prefix="{{:: 'dedicated::' + $ctrl.trackingPrefix }}"
+                tracking-prefix="DedicatedServers::dedicated-server::node"
```

Comment:
I'm not sure this is correct, I think this component is used for 'dedicated-servers', 'dedicated-cluster' and 'dedicated-node'